schematron/v2.1.0/exemplary_files/L100_Audit.xml

```diff
@@ -0,0 +1,766 @@
+                  <auc:EndTimestamp>2019-09-01T00:00:00</auc:EndTimestamp>
+                  <auc:IntervalFrequency>Month</auc:IntervalFrequency>
+                  <auc:IntervalReading>123</auc:IntervalReading>
+                  <auc:ResourceUseID IDref="ResourceUse-NG" />
+                </auc:TimeSeries>
+                <auc:TimeSeries ID="Scenario-A-NG-TS9">
+                  <auc:ReadingType>Total</auc:ReadingType>
+                  <auc:StartTimestamp>2019-09-01T00:00:00</auc:StartTimestamp>
+                  <auc:EndTimestamp>2019-10-01T00:00:00</auc:EndTimestamp>
+                  <auc:IntervalFrequency>Month</auc:IntervalFrequency>
+                  <auc:IntervalReading>123</auc:IntervalReading>
+                  <auc:ResourceUseID IDref="ResourceUse-NG" />
+                </auc:TimeSeries>
+                <auc:TimeSeries ID="Scenario-A-NG-TS10">
+                  <auc:ReadingType>Total</auc:ReadingType>
+                  <auc:StartTimestamp>2019-10-01T00:00:00</auc:StartTimestamp>
+                  <auc:EndTimestamp>2019-11-01T00:00:00</auc:EndTimestamp>
+                  <auc:IntervalFrequency>Month</auc:IntervalFrequency>
+                  <auc:IntervalReading>123</auc:IntervalReading>
+                  <auc:ResourceUseID IDref="ResourceUse-NG" />
+                </auc:TimeSeries>
+                <auc:TimeSeries ID="Scenario-A-NG-TS11">
+                  <auc:ReadingType>Total</auc:ReadingType>
+                  <auc:StartTimestamp>2019-11-01T00:00:00</auc:StartTimestamp>
+                  <auc:EndTimestamp>2019-12-01T00:00:00</auc:EndTimestamp>
+                  <auc:IntervalFrequency>Month</auc:IntervalFrequency>
+                  <auc:IntervalReading>123</auc:IntervalReading>
+                  <auc:ResourceUseID IDref="ResourceUse-NG" />
+                </auc:TimeSeries>
+                <auc:TimeSeries ID="Scenario-A-NG-TS12">
+                  <auc:ReadingType>Total</auc:ReadingType>
+                  <auc:StartTimestamp>2019-12-01T00:00:00</auc:StartTimestamp>
+                  <auc:EndTimestamp>2020-01-01T00:00:00</auc:EndTimestamp>
+                  <auc:IntervalFrequency>Month</auc:IntervalFrequency>
+                  <auc:IntervalReading>123</auc:IntervalReading>
+                  <auc:ResourceUseID IDref="ResourceUse-NG" />
+                </auc:TimeSeries>
+              </auc:TimeSeriesData>
+
+              <!-- 6.1.2.2 -->
+              <auc:AllResourceTotals>
+                <auc:AllResourceTotal ID="AllResourceTotal-A">
+                  <auc:SiteEnergyUse>123</auc:SiteEnergyUse>
+                  <auc:SiteEnergyUseIntensity>123</auc:SiteEnergyUseIntensity>
+                  <auc:EnergyCost>123</auc:EnergyCost>
+                  <auc:EnergyCostIndex>123</auc:EnergyCostIndex>
+                </auc:AllResourceTotal>
+              </auc:AllResourceTotals>
+            </auc:Scenario>
+
+            <!-- Current energy use -->
+            <auc:Scenario ID="Scenario-B">
```

Comment:
@macintoshpie not sure what this scenario represents...seems duplicate to a 'current building measured', i.e. to `Scenario-A` right above it?